OWML.Launcher/App.cs

```diff
@@ -70,6 +72,54 @@ private void LocateGamePath()
             }
         }
 
+        private void CheckGameVersion()
+        {
+            var versionReader = new GameVersionReader(_writer, new BinaryPatcher(_owmlConfig, _writer));
+            var gameVersion = versionReader.GetGameVersion();
+            _writer.WriteLine($"Game version: {gameVersion}", MessageType.Info);
+            var splitGameVersion = gameVersion.Split('.');
+            var splitMinVersion = _owmlManifest.MinimalGameVersion.Split('.');
+            var splitMaxVersion = _owmlManifest.MaximalGameVersion.Split('.');
+            var count = Math.Min(splitGameVersion.Length, Math.Min(splitMaxVersion.Length, splitMinVersion.Length));
+            splitGameVersion[count-1] = string.Join("", splitGameVersion.Skip(count - 1));
+            splitMinVersion[count-1] = string.Join("", splitMinVersion.Skip(count - 1));
+            splitMaxVersion[count-1] = string.Join("", splitMaxVersion.Skip(count - 1));
```

Comment:
not really sure about that part TBH. It feels like it might backfire in some way